lib/iex/resources.rb

```diff
@@ -13,3 +13,4 @@
 require_relative 'resources/quote'
 require_relative 'resources/sectors'
 require_relative 'resources/crypto'
+require_relative 'resources/isin_mapping'
```

Comment:
`resources/isin` to be consistent with the API

Reply:
`Helper call to convert ISIN to IEX Cloud symbols` from https://iexcloud.io/docs/api/#isin-mapping. Maybe it's better to call the resource `symbol`? WDYT?

Reply:
I worry about something so generic as `symbol`, but I'm OK with it. Your call, it does return a symbol with its metadata.

Generally the client should not be too opinionated. If IEX calls something a certain way we should stick to it.

Reply:
In this case, should we have two endpoints like `isin` and `isin_mapped` if IEX returns different responses depending on the request options?